### PR TITLE
Nonpr 2242

### PR DESCRIPTION
**Pull request by Leeroyc720**
```
[info] Run completed in 14 seconds, 968 milliseconds.
[info] Total number of tests run: 170
[info] Suites: completed 21, aborted 0
[info] Tests: succeeded 170, failed 0, canceled 0, ignored 0, pending 0
[info] All tests passed.
[info] Passed: Total 170, Failed 0, Errors 0, Passed 170
[success] Total time: 30 s, completed 27 Sep 2021, 14:10:36
```

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -216,6 +216,18 @@ notableEvents = [
...
   },
+  {
+    notableEvent = "itsa-cgt-disposal"
+    displayName = "ITSA CGT Residential Property Disposal",
+    storedFrom = "18th Oct 2021",
+    storedFor = "7 years",
+    searchKeys = [
+      {
+        name = "nino",
+        label = "National Insurance Number (NINO)"
```
Comment (craigtd) [inline conf/application.conf:227]: @Leeroyc720 I have just noticed a discrepancy with what is in https://confluence.tools.tax.service.gov.uk/display/NR/Calling+Service+Registry (for this and all the earlier ITSA notable events - which addmittedly was my error). This label should just be `NINO` - which I think should be good enough for the UI. What do you think?

Comment (craigtd) [reply conf/application.conf:227]: I think it is safe enough to assume that FIS and everyone working for HMRC knows what a NINO is

Comment (Leeroyc720) [reply conf/application.conf:227]: I think that's fine as you have `Vat Registration Number (VRN) ` it seems the thought was to be consistent with the other services such as VAT, if we are changing this we would change this for VAT too?

Comment (craigtd) [reply conf/application.conf:227]: The problem is that the calling service registry captures what the calling service wanted us to display. The VAT team specified the expanded name, whereas ITSA specified just NINO.

Comment (Leeroyc720) [reply conf/application.conf:227]: Ok that's fine, I will change them for ITSA

**Review by craigtd — APPROVED** [review]
all good

**Pull request merged**